build/config/android/config.gni

```diff
@@ -287,7 +287,8 @@ if (is_android || is_chromeos) {
     enable_incremental_d8 = true
 
     # Use hashed symbol names to reduce JNI symbol overhead.
-    use_hashed_jni_names = !is_java_debug
+    # Cobalt avoids hash names to facilitate Kimono build.
+    use_hashed_jni_names = false
```

Comment:
perhaps leave it `use_hashed_jni_names = !is_java_debug || is_cobalt` to ensure Chromium build configs don't change here.

Reply:
This PR pretty much just provides two gn flags to work with Kimono build. I'm just sharing my workspace with Andrew to figure out some build problems for now. This PR is far from review yet.